renderdoc/driver/d3d12/d3d12_device_wrap.cpp

```diff
@@ -1266,7 +1266,7 @@ void WrappedID3D12Device::CopyDescriptors(
                              NumSrcDescriptorRanges, srcStarts, pSrcDescriptorRangeSizes,
                              DescriptorHeapsType);
 
-  RDCUNIMPLEMENTED("CopyDescriptors does not copy our internal descriptor data");
+	D3D12NOTIMP("CopyDescriptors does not copy our internal descriptor data");
```

Comment:
Yes this was just leftover by accident when I implemented the function, it can be deleted! :smile: